src/test/test_bitcoin.cpp

```diff
@@ -31,6 +31,8 @@ std::ostream& operator<<(std::ostream& os, const uint256& num)
     return os;
 }
 
+extern void noui_connect_noop_handlers();
```

Comment:
No need for extern and I meant to move the whole function body. There shouldn't be need to modify the binary for fixing up tests.

Reply:
Good point. Now touching only `src/test/` and `src/wallet/test/`. Please re-re-review :-)